specs/MediaObject.md

```diff
@@ -0,0 +1,75 @@
+MediaObject
+===========
+
+A layout utility for aligning a media figure next to text.
+
+Inspiration
+-----------
+
+* https://getbootstrap.com/docs/4.0/layout/media-object/
+* https://lightningdesignsystem.com/utilities/media-objects/
+* https://philipwalton.github.io/solved-by-flexbox/demos/media-object/
+* http://www.stubbornella.org/content/2010/06/25/the-media-object-saves-hundreds-of-lines-of-code
+
+Discovery
+---------
+
+#### Are there similar components already in the library? If yes, what are they and can they be modified instead to capture this use-case?
+
+* No
+
+#### Who is the primary consumer for this component? Does it belong in the core API?
+
+* Core component
+
+#### Does the component behave any differently on desktop than it does on mobile? Is there any touch specific behavior?
+
+* No
+
+#### How does the component behave with respect to other components?
+
+* Should support any content for the body and figure. Should allow nesting of other `MediaObject` components.
+
+#### Are there any existing libraries that can potentially be leveraged?
+
+* No
+
+#### Are there any performance considerations?
+
+* No
+
+#### [Theming] Is there any animation, transition, or other forms of motion involved?
+
+* No
+
+#### [Theming] Are there any z-index considerations?
+
+* No
+
+Accessibility
+-------------
+
+* Should we use the HTML [`<figure>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure) element?
```

Comment:
This I'm not sure about... any thoughts?

Reply:
Could it optionally have a caption?  If so, I would say go with figure